### PR TITLE
Djm/pkgdown updates

### PR DESCRIPTION
**Pull request by dajmcdon**
Changes are in the commit messages.

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
```
Comment (dshemetov) [inline README.Rmd:5]: I was hoping there was a way to generate the README from an rmd, thanks!

Comment (dshemetov) [reply README.Rmd:5]: Does `devtools::document()` or `check` automatically run `devtools::render_readme()`?

Comment (dajmcdon) [reply README.Rmd:5]: There should be an action to do this automatically, but I couldn't find it. I know I've seen it used elsewhere though.

Comment (dajmcdon) [reply README.Rmd:5]: Not that I know of.

File: README.md [hunk 2]
```diff
@@ -1,18 +1,47 @@
...
+[![codecov](https://codecov.io/gh/dsweber2/epidatr/branch/dev/graph/badge.svg?token=jVHL9eHZNZ)](https://app.codecov.io/gh/dsweber2/epidatr)
+<!-- badges: end -->
+
+The [Delphi Epidata API](https://cmu-delphi.github.io/delphi-epidata/)
+provides real-time access to epidemiological surveillance data for
+influenza, COVID-19, and other diseases for the USA at various
+geographical resolutions, both from official government sources such as
+the [Center for Disease Control
+(CDC)](https://www.cdc.gov/datastatistics/index.html) and [Google
+Trends](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/google-symptoms.html)
```
Comment (dshemetov) [inline README.md:21]:
```suggestion
geographical resolutions, both from official government sources such as
the [Center for Disease Control
(CDC)](https://www.cdc.gov/datastatistics/index.html), public sources such as [Google
Trends](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/google-symptoms.html),
```

File: README.Rmd (hunk 1 above)
Comment (dshemetov) [reply README.Rmd:5]: We have a handy Makefile with some chore commands, I guess in absence of something R-built-in, we use that

Comment (dshemetov) [reply README.Rmd:5]: Ah yea, looks like [they noped out of doing it auto](https://github.com/r-lib/devtools/issues/2330)

Comment (dshemetov) [reply README.Rmd:5]: Oh in a comment there, found `usethis::use_readme_rmd()`, but it's somewhat interactive (and writes a git pre-commit hook)

**Pull request merged**